bears/general/AnnotationBear.py

```diff
@@ -18,14 +18,17 @@ def run(self, filename, file, language: str, coalang_dir: str = None):
         The Bear searches for valid comments and strings and yields their
         ranges as SourceRange objects in HiddenResults.
 
-        :param language:        Language to be whose annotations are to be
-                                searched.
-        :param coalang_dir:     external directory for coalang file.
-        :return:                HiddenResults containing a dictionary with
-                                keys as 'strings' or 'comments' and values as
-                                a tuple of SourceRanges of strings and
-                                a tuple of SourceRanges of comments
-                                respectively.
+        :param language:
+            Language to be whose annotations are to be searched.
```

Comment:
Either my english module is malfunctioning or this sentence is broken.
How about `Determines the languages (and therefore comment styles) to expect  .` or something?